packages/pigeon/lib/cpp_generator.dart

```diff
@@ -2,10 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
+import 'package:path/path.dart' as path;
+
 import 'ast.dart';
 import 'functional.dart';
+import 'generator.dart';
 import 'generator_tools.dart';
-import 'pigeon_lib.dart' show Error;
+import 'pigeon_lib.dart' show Error, PigeonOptions, lineReader, openSink;
```

Comment:
We should ideally move these two utility methods to a utility file, and find somewhere for the other things to live, so we don't have circular includes. Than can be done later though.